cmake/onnxruntime_webassembly.cmake

```diff
@@ -268,7 +268,7 @@ else()
   endif()
 
   if (onnxruntime_USE_WEBNN)
-    set_property(TARGET onnxruntime_webassembly APPEND_STRING PROPERTY LINK_FLAGS " --bind -sWASM_BIGINT")
+    set_property(TARGET onnxruntime_webassembly APPEND_STRING PROPERTY LINK_FLAGS " --bind -sWASM_BIGINT -sASYNCIFY=1 -sASYNCIFY_STACK_SIZE=65536")
```

Comment:
I need an answer for this question: do we have the requirement to build webnn without webgpu?

If the answer is 'NO', we need to enforce build flags to make sure when building webnn, JSEP is always on. In this case, we don't need to put this flag here.

If the answer is 'YES', we need to split js_internal_api.js into 3 parts: the shared, webnn and webgpu. build flags, CI builds also need to split into multiple

Reply:
Thanks @fs-eire!

> I need an answer for this question: do we have the requirement to build webnn without webgpu?

I am not aware any requirements to build webnn without webgpu. Building webnn with JSEP makes sense to me. 

Longer term, FYI, WG is driving webnn / webgpu interop. When it is supported, webnn may fallback to webgpu. And webnn EP may also change to JSEP (thoughts?). Having them together might be more future proof.